packages/nextalign/src/nextalign.cpp

```diff
@@ -31,6 +31,18 @@ RefPeptide toRefPeptideExternal(const RefPeptideInternal& peptide) {
   return RefPeptide{.name = peptide.geneName, .seq = toString(peptide.peptide)};
 }
 
+std::string formatInsertion(const NucleotideInsertion& insertion) {
+  // NOTE: by convention, in bioinformatics, nucleotides are numbered starting from 1, however our arrays are 0-based
+  const auto positionOneBased = insertion.pos + 1;
+  const auto insertedSequence = toString(insertion.ins);
+  return fmt::format("{}:{}", positionOneBased, insertedSequence);
+}
+
+std::string formatInsertions(const std::vector<NucleotideInsertion>& insertions) {
+  return formatAndJoin(insertions, formatInsertion, ";");
+}
```

Comment:
Fix in (C): The deduplicated functions to format insertions for CSV output. Shared between Nextalign, Nextclade CLI and Nextclade Web (through WebAssembly)